apps/admin/frontend/src/components/import_cvrfiles_modal.tsx

```diff
@@ -62,6 +65,89 @@ const Content = styled.div`
   overflow: hidden;
 `;
 
+/* c8 ignore start */
+function userReadableMessageFromError(
```

Comment:
Note that this function lives on the frontend. The old import code has an equivalent function live on the backend and sends a user-readable message down to the frontend for it to use. Ultimately, this message-preparation is a UI/frontend concern and not a backend concern, hence my moving it to the frontend.

This will become even more necessary once we start working on multi-language support (as noted [here](https://docs.google.com/document/d/1a2GRj1g4aOawEssPdTCgbv-qs28d-M_dyMEfzm4I4nM/edit#heading=h.gno6ffrtd5t)).